packages/server/src/sdk/app/views/index.ts

```diff
@@ -252,3 +252,49 @@ export function syncSchema(
 
   return view
 }
+
+export async function renameLinkedViews(table: Table, renaming: RenameColumn) {
+  const relatedTables: Record<string, Table> = {}
+
+  for (const field of Object.values(table.schema)) {
+    if (field.type !== FieldType.LINK) {
+      continue
+    }
+
+    relatedTables[field.tableId] ??= await sdk.tables.getTable(field.tableId)
+    break
```

Comment:
I don't fully understand the purpose of this `break`. Wouldn't you want to fetch all related tables?

Reply:
Good catch! This is left from a initial implementation that was behaving differently. I will check why the test I wrote did not catch that :/

Reply:
Tests added